src/xl/tables.ts

```diff
@@ -1,5 +1,5 @@
-export const DATA_SCIENCE_EXPORT_SHEET = "Data Science Editor Export";
-export const DATA_SCIENCE_EXPORT_TABLE = "DataScienceEditorTable";
+export const dataScienceExportSheet = "Data Science Editor Export";
```

Comment:
I like to capitalize constants, call me old school. Disable the rule.

Reply:
fixed.